### ISSUE TITLE
Is my gulp giving me a valid result? or is something wrong

### ISSUE THREAD

**Issue by msdickinson** [asu-ke-web-services/dont-get-fired-game#6]
What I did. (Was creating a guide for my self)
1. https://git-for-windows.github.io/ Download, Install (On Line Endings do Cehc out as-is, commit Unix-Style line endings) 
2. I am not sure if you have to this step. https://nodejs.org/en/ Download Install (I Did do this)
3. Open Git CMD as Admin
4. Type "cd sustainability-game"
5. Type" git clone  http://github.com/gios-asu/sustainability-game.git" into Git CMD and press enter.
6. Copy and paste from https://github.com/gios-asu/sustainability-game
   "npm install -g gulp
   npm install -g browserify
   npm install
   gulp" into the Git CMD
   7.if gulp doesnt run press enter.

When I Gulp, I get this result.

![isthiserror](https://cloud.githubusercontent.com/assets/2480828/10735941/ae23b6b6-7bc7-11e5-866e-39c3cb685302.PNG)

Here is my full log

[Log.txt](https://github.com/gios-asu/sustainability-game/files/21061/Log.txt)

**Comment by idmontie** [comment]
I am looking into this. #8 might fix it once it is merged into develop.

**Comment by idmontie** [comment]
Run `git pull origin develop` to pull the `develop` branch into your branch.

**Comment by idmontie** [comment]
Then run `gulp` and let me know what it outputs. Thanks.

**Comment by msdickinson** [comment]
same for me.
![testthis](https://cloud.githubusercontent.com/assets/2480828/10738682/b443f21e-7bd5-11e5-89f1-0a8c09e52c31.PNG)

Is this related?
https://github.com/gios-asu/sustainability-game/branches -> Mine is Mark and has a X

**Comment by idmontie** [comment]
Ah, I remember this now. Coveralls if failing for some reason on your machine. 

For now, just comment out the following lines in in `gulpfile.js` (but don't commit this change):

https://github.com/gios-asu/sustainability-game/blob/develop/gulpfile.js#L118-L121

and

https://github.com/gios-asu/sustainability-game/blob/develop/gulpfile.js#L123-L126

Not sure why coveralls is failing on your Windows machine, it works fine on my Windows machine. We'll look into it more during our meeting.

**Comment by idmontie** [comment]
Can you open up `C:\Users\mdickinson\School\sustainability-game` and see if there is a folder `debug` in there?

**Comment by msdickinson** [comment]
after removing those it worked. fine.

There is no Debug Folder
![nodebug](https://cloud.githubusercontent.com/assets/2480828/10739813/e6b04300-7bdb-11e5-960a-da4a29bd6d82.PNG)

**Comment by idmontie** [comment]
My guess is that gulp/coveralls is unable to make the debug directory. If it can't make it, it can't move files there.
